cmd/snap/cmd_snap_op.go

```diff
@@ -1209,6 +1230,41 @@ func (x *cmdRefresh) Execute([]string) error {
 	return x.refreshMany(names, opts)
 }
 
+func (x *cmdRefresh) TrackRefreshes() (err error) {
```

Comment:
This doesn't need to be exported:

```suggestion
func (x *cmdRefresh) trackRefreshes() (err error) {
```